GithubUsers-Redux/States/UserListState.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UserListState.swift
+//  GithubUsers-Redux
+//
+//  Created by Shabir Jan on 3/24/18.
+//  Copyright © 2018 Shabir Jan. All rights reserved.
+//
+
+import Foundation
+
+struct UserListState {
+    let users: [User]
```

Comment:
that means that it's only one possible state and this is when you have an array of users, but there is also a state when you don't like `loading`

Reply:
what would that be?